src/page/moments.ts

```diff
@@ -88,13 +88,31 @@ export default class Moments {
    * @return {*}
    * @param {*}
    */
-  @documentFunction(false)
+  @documentFunction()
   public registerMomentItem(containerElement?: HTMLElement) {
     const momentContainerElement = containerElement || document.querySelector(".moments-container .moments-inner");
     if (!momentContainerElement) {
       return;
     }
-    const momentItemElements = momentContainerElement?.querySelectorAll(".moments-item") as NodeListOf<HTMLElement>;
+    
+    const existItems = momentContainerElement.querySelectorAll(".moments-item");
+    existItems.forEach(item => {
+      const likeBtn = item.querySelector(".moment-tools .moment-like");
+      const commentBtn = item.querySelector(".moment-tools .comment-js");
+      
+      if (likeBtn) {
+        const newLikeBtn = likeBtn.cloneNode(true);
+        likeBtn.parentNode?.replaceChild(newLikeBtn, likeBtn);
+      }
+      
+      if (commentBtn) {
+        const newCommentBtn = commentBtn.cloneNode(true);
+        commentBtn.parentNode?.replaceChild(newCommentBtn, commentBtn);
+      }
+    });
+    
```

Comment:
```suggestion
    const existItems = momentContainerElement.querySelectorAll(".moments-item");
    existItems.forEach(item => {
      const likeBtn = item.querySelector(".moment-tools .moment-like");
      const commentBtn = item.querySelector(".moment-tools .comment-js");
      
      if (likeBtn) {
        const newLikeBtn = likeBtn.cloneNode(true);
        likeBtn.parentNode?.replaceChild(newLikeBtn, likeBtn);
      }
      
      if (commentBtn) {
        const newCommentBtn = commentBtn.cloneNode(true);
        commentBtn.parentNode?.replaceChild(newCommentBtn, commentBtn);
      }
    });
```

这块代码我不是特别理解。移除再重新嵌入的作用是什么？？

如果仅仅是解决 pjax 下点赞和评论功能的问题，只需要使  `@documentFunction(false)` 中去掉 false 让每次 pjax 执行后都执行此方法即可。

Reply:
我第一次就是去掉了 `@documentFunction(false)` 的 `false`，但是发现加载更多后评论框无法展开，所以我就尝试了克隆替换。

Reply:
> 我第一次就是去掉了 `@documentFunction(false)` 的 `false`，但是发现加载更多后评论框无法展开，所以我就尝试了克隆替换。

了解了。但感觉不是特别合适，可以说规避了问题但没有修复。

我发现可能还不仅仅是这个问题，原因在于上面的 DOMParser 被多次解析处理，进而导致其内部的方法 `sakura.refresh()` 被多次执行，多次执行后就会导致异常问题。至于为什么会多次执行暂且未知。

Reply:
嗯。。发现问题了，执行 `sakura.refresh()` 之后会导致分页事件重复执行，进行会出现循环执行的问题，我会修复一下并推送到当前 PR。

Reply:
> > 我第一次就是去掉了 `@documentFunction(false)` 的 `false`，但是发现加载更多后评论框无法展开，所以我就尝试了克隆替换。
> 
> 了解了。但感觉不是特别合适，可以说规避了问题但没有修复。
> 
> 我发现可能还不仅仅是这个问题，原因在于上面的 DOMParser 被多次解析处理，进而导致其内部的方法 `sakura.refresh()` 被多次执行，多次执行后就会导致异常问题。至于为什么会多次执行暂且未知。

哦哦，我没怎么深入学习过，深层问题我就不太了解，所以我就只能遇水架桥，暂时解决眼前的问题。

Reply:
> 嘿嘿，好的，祝大佬假期玩得愉快。

不敢当不敢当。祝你假期也玩的开开心心。

代码我提交了，你可以抽空测试一下看看问题是否已经解决了。